### DISCUSSION TITLE
Allow adding to supported chains

### DISCUSSION THREAD

**Discussion by dirtycajunrice** [family/connectkit#19] [Ideas]
The current implementation does not allow configuring supported chains. Although wagmi allows it, and connectkit will connect to it, logos cant be set.

**Comment by lochie** [comment]
Hey @dirtycajunrice,

We do currently allow custom chains to be configured just like you would in wagmi ([see docs](https://docs.family.co/connectkit/chains)), but indeed the chain logo we haven't made customisable yet, but we're looking to get this in within one of the next few updates!

**Comment by royalaid** [comment]
Just curious if this being worked out. If not it seems like the PR would be pretty simple and I might be able to take a stab
